example/bottle/server.py

```diff
@@ -0,0 +1,34 @@
+from bottle import route, run, app, static_file
+import bugsnag
+
+from bugsnag.wsgi.middleware import BugsnagMiddleware
+
+bugsnag.configure(
+    api_key='698ffb05632f0ffcad3ebd16383507fd',
+)
+
+@route('/')
+def index():
+    return static_file('index.html', root='.')
+
+@route('/crash')
+def crash():
+    """Deliberately raises an unhandled error and crash the app.
+    """
+    raise(Exception('SomethingBad'))
```

Comment:
raise is not a function

```suggestion
    raise Exception('SomethingBad')
```

also flake8 is failing on this file:

```
$ flake8 example/bottle/server.py
example/bottle/server.py:10:1: E302 expected 2 blank lines, found 1
example/bottle/server.py:14:1: E302 expected 2 blank lines, found 1
example/bottle/server.py:20:1: E302 expected 2 blank lines, found 1
example/bottle/server.py:25:9: F841 local variable 'x' is assigned to but never used
example/bottle/server.py:25:14: E226 missing whitespace around arithmetic operator
example/bottle/server.py:29:80: E501 line too long (93 > 79 characters)
example/bottle/server.py:31:1: E305 expected 2 blank lines after class or function definition, found 1
```